core/store/migrate/migrations/0086_ocr2_tables.sql

```diff
@@ -2,27 +2,27 @@
 -- +goose StatementBegin
 CREATE TABLE offchainreporting2_oracle_specs (
```

Comment:
This migration has already done into develop so we shouldn't be changing it as anyone running develop will have an up to date schema and these changes will not be applied

Reply:
No one will be using ocr2 jobs yet and doing it this way keeps the migration a lot more readable. I'm ok to keep it separate to not for a local redeploy for folks but not sure its worth the readability hit?

Reply:
spoke offline - will update